platforms/cpuinfo.go

```diff
@@ -43,6 +45,32 @@ func cpuVariant() string {
 	return cpuVariantValue
 }
 
+// get CPU Architecture from uname -m
+func getCPUArchitecture() (string, error) {
+	if !isLinuxOS(runtime.GOOS) {
+		return "", fmt.Errorf("getCPUArchitecture for OS %s: %w", runtime.GOOS, errdefs.ErrNotImplemented)
+	}
+	var name unix.Utsname
+	err := unix.Uname(&name)
+	if err != nil {
+		return "", err
+	}
+	machine := string(name.Machine[:bytes.IndexByte(name.Machine[:], 0)])
+	// https://github.com/qemu/qemu/blob/master/linux-user/uname.c
+	if machine == "aarch64" {
+		return machine, nil
+	}
```

Comment:
Why does it need to handle aarch64 here?

Reply:
aarch64 are used alternatively with armv8. It doesn't follow the name convention as armv7, armv6. So we need to handle this separately.  
We return it directly as it will later be [normalized](https://github.com/containerd/containerd/blob/main/platforms/cpuinfo.go#L127).